specs/chain/latex/blockchain-spec.tex

```diff
@@ -1306,14 +1310,17 @@ \section{Blockchain Extension}
   such an invalid chain being produced by the old procedure of randomly selecting
   the slot leaders for each slot. Given the Cardano chain is still federated, the
   likelihood of this happening is the same for each of the 7 stakeholders, and we
-  may model the number of selected slots within a $w$-slot window $X$ as a binomial
-  distribution $X \sim \mathrm{B}\left(w, \frac{1}{7}\right)$.
+  may model the number of selected slots within a $k$-slot window $X$ as a binomial
+  distribution $X \sim \mathrm{B}\left(k, \frac{1}{7}\right)$.
 
-  In each epoch of size $n$ blocks, there are $n-w+1$ such $w$-block windows.
+  In each epoch of size $n$ blocks, there are $n-k+1$ such $w$-block windows.
```

Comment:
Should this be "$k$-block windows"?

Reply:
Right! I forgot to change that one. Thanks!